.eslintrc

```diff
@@ -7,7 +7,8 @@
     "env": {
         "browser": true,
         "node": true,
-        "es6": true
+        "es6": true,
+        "mocha": true
```

Comment:
you can add another `.eslintrc` file inside the `test/` dir to make this `mocha` thing only apply to files in that dir.